app/assets/javascripts/backbone/views/timeline_table_view.coffee

```diff
@@ -5,8 +5,14 @@ class MS.Views.TimelineTableView extends Backbone.View
 
   initialize: (options) ->
     new MS.Views.AutoLoadingIndicatorView()
-    @loanId = options.loanId
-    @groupModal = new MS.Views.ProjectGroupModalView(loanId: @loanId, success: @refresh.bind(@))
+    @projectId = options.projectId
+
+    if options.projectType == 'BasicProject'
```

Comment:
Yes, coffeescript doesn't like the `? :` operator. You can use `if` `else` on one line instead but this if fine.